src/ZODB/tests/test_fsdump.py

```diff
@@ -58,7 +58,7 @@
 Trans #00001 tid=... time=... offset=<OFFSET>
     status=' ' user='' description='added an OOBTree'
   data #00000 oid=0000000000000000 size=<SIZE> class=persistent.mapping.PersistentMapping
-  data #00001 oid=0000000000000001 size=<SIZE> class=BTrees.OOBTree.OOBTree
+  data #00001 oid=0000000000000001 size=<SIZE> class=BTrees.OOBTree.OOBTree...
```

Comment:
Under CPython, class is as you see it. Under PyPy or PURE_PYTHON, class is `OOBTreePy`.